beetsplug/subsonicupdate.py

```diff
@@ -78,10 +86,26 @@ def loaded(self):
             'v': '1.15.0',  # Subsonic 6.1 and newer.
             'c': 'beets'
         }
+
+        if port == 443:
+            schema = 'https://'
+
         if contextpath == '/':
             contextpath = ''
-        url = "http://{}:{}{}/rest/startScan".format(host, port, contextpath)
+
+        if not url:
+            url = "{}{}:{}{}/rest/startScan" \
+                .format(schema, host, port, contextpath)
```

Comment:
Instead of doing this here, let's instead put something like this up near the initial configuration setup stuff:

``py
if not url:
    host = config['subsonic']['host'].as_str()
    # et cetera
    url = "http://{}:{}{}".format(host, port, contextpath)
```

Then, down here, we can just use `url` knowing that it is populated with the right thing.